pyproject.toml

```diff
@@ -102,15 +101,15 @@ graphene-django-optimizer = "~0.8.0"
 # Template rendering engine
 Jinja2 = "~3.1.6"
 # Optional data validation of config contexts - loose dependency since our usage is light but apps may be more specific
-jsonschema = "^4.7.0"
+jsonschema = "^4.25.0"
```

Comment:
Do we want to raise the lower bound like this, or keep it where it's at as a loose constraint?